Week2/akif.js

```diff
@@ -0,0 +1,108 @@
+// Part 1 Hello World
+console.log('Hello World'); // English
+console.log('Merhaba Dunya'); // Turkish
+console.log('Hallo Wereld'); // Dutch
+console.log('مرحبا بالعالم'); // Arabic
+
+// Part 2 Error  -----------------------------------------
+console.log("I'm awesome");
+
+// Part 3 Console.log(x)   --------------------------------
+let x;
+console.log('the value of my variable x will be: numberOfTheStudents');
```

Comment:
You were supposed to replace `numberOfTheStudents` with the value which you expect it should have been i.e `undefined` or `16` or something like that. You should be doing it before you run the code in browser or VS Code terminal. This is needed to ensure that you understand how JS works. 🙂 

Can you please correct it?